src/types.ts

```diff
@@ -43,10 +43,17 @@ export type WeekRowsType = {
   cells: WeekCellType[][]
 }
 
+export type DayRowsType = {
+  time: string
+  cells: DayCellType[][]
+}
+
 export type WeekCellType = CalendarEventType & {
   duration?: Duration
 }
 
+export type DayCellType = WeekCellType
+
```

Comment:
What is the point?

Maybe it's going to be better if you give it a general name that is going to suit both?